dali/test/python/decoder/test_imgcodec.py

```diff
@@ -99,94 +99,79 @@ def test_image_decoder():
 
 
 @pipeline_def
-def create_decoder_slice_pipeline(data_path, device, is_roi_dec):
+def create_decoder_slice_pipeline(data_path, device):
```

Comment:
this is reverting the changes from https://github.com/NVIDIA/DALI/commit/b44a50778a69caadbdf4a46b4491a485573af747#diff-08654402c88b7fe5e35a100dad2282f7b139c6643d65c00c78e72bccde6fcdc5